package.json

```diff
@@ -39,10 +50,7 @@
   },
   "scripts": {
     "typecheck": "tsc --noEmit",
-    "build:cjs": "tsc -p ./tsconfig.build-cjs.json",
-    "build:esm": "tsc -p ./tsconfig.build-esm.json",
-    "build:types": "tsc -p ./tsconfig.build-types.json",
-    "build": "rimraf dist && concurrently \"npm run build:types\" \"npm run build:esm\" \"npm run build:cjs\"",
+    "build": "tsup --config ./tsup.config.ts",
```

Comment:
[question] Early thoughts from `tsup` usage?

Reply:
It's too early to say. Looks good so far.